olive/passes/onnx/quantization.py

```diff
@@ -417,16 +418,21 @@ def _run_for_config(
         # load the model
         onnx_model = onnx.load(tmp_model_path)
         # the model is loaded into memory, so it's safe to delete previously exported files
-        tmp_dir.cleanup()
+        self.tmp_dir.cleanup()
 
         # save the model to the output path and return the model
         return model_proto_to_olive_model(onnx_model, output_model_path, config)
 
-    def _quant_preprocess(self, model: ONNXModel, output_model_path: str) -> ONNXModel:
+    def _quant_preprocess(self, model: ONNXModel, output_model_path: Union[str, Path]) -> ONNXModel:
         from onnxruntime.quantization.preprocess import quant_pre_process
 
         try:
-            quant_pre_process(input_model_path=model.model_path, output_model_path=output_model_path, auto_merge=True)
+            quant_pre_process(
+                input_model_path=model.model_path,
+                output_model_path=str(output_model_path),
+                auto_merge=True,
+                save_as_external_data=True,
```

Comment:
This one makes sense since it is better to `save_as_external_data` even if not needed than for it to fail for large models. 

IIRC, i had some issues setting this to true always because of issues when loading the model.

Reply:
I had similar problems and as it turns out it was because of the missing `str` for otput_model_path. `onnxruntime.save_model` doesn't support `Path` object and so the argument was not being resolved correctly.

Reply:
Oh I see. Thanks!